apps/dashboard/app/controllers/projects_controller.rb

```diff
@@ -73,6 +74,7 @@ def update
 
   # POST /projects
   def create
+    params[:project][:icon] = icon_with_uri(params[:project][:icon])
```

Comment:
I still don't think we should be editing `params` inline. 

Seems like we should push this logic down to the model. Maybe as a before_validation? 

https://api.rubyonrails.org/v7.1.3.4/classes/ActiveModel/Validations/Callbacks/ClassMethods.html#method-i-before_validation

https://github.com/OSC/ondemand/blob/1498f09b6c4ee3e3d17f62f22ce154ace39a401d/apps/dashboard/app/models/project.rb#L72